.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -33,31 +33,21 @@ body:
 
   # Files Version
   - type: input
-    id: files-version
+    id: files_version
     attributes:
       label: Files Version
-      description: Which version of Files are you currently using?
-      placeholder: 'Version: 2.0.36.0'
+      description: Which version of Files are you using? To copy your Files version, access it from Settings -> About -> Copy -> Files version
+      placeholder: 'e.g. 2.0.36.0'
     validations:
       required: true
 
   # Windows Version
-  - type: dropdown
+  - type: input
     id: windows_version
     attributes:
       label: Windows Version
-      description: Which version of Windows are you using?
-      multiple: true
-      options:
-        - 'Windows 11 Insider: 25xxx series'
-        - 'Windows 11 Insider: 23xxx series'
-        - 'Windows 11 (22H2): 22621'
-        - 'Windows 11 (21H2): 22000'
-        - 'Windows 10 (22H2): 19045'
-        - 'Windows 10 (21H2): 19044'
-        - 'Windows 10 (21H1): 19043'
-        - 'Windows 10 (20H2): 19042'
-        - 'Windows 10 (2004): 19041'
+      description: Which version of Windows are you using? To copy your Windows version, access it from Settings -> About -> Copy -> Windows version
+      placeholder: "e.g. 10.0.22621.1848"
```

Comment:
Can you do without changing windows version input type from dropdown, so users who didn't use Files' settings report buttons can easily set a value?

Reply:
Thanks for the review. I tried keeping the dropdown, but as I tested, the dropdown and checkbox in the issue form doesn't prefill, and I found it is existing issue: 
https://github.com/orgs/community/discussions/44983
https://github.com/orgs/community/discussions/42493

How do you think about adding a sentence in issue form description to recommend users create issue via the About Page. For example:
![image](https://github.com/files-community/Files/assets/26683979/061e16f6-9a2f-445b-9ad0-b71b26584fb6)

If they decide not to do that, they might follow the instructions on how to copy the Windows version:
https://github.com/files-community/Files/blob/d1906c032f89931e20c1d1688067339a3f5799f0/.github/ISSUE_TEMPLATE/feature_request.yml?plain=1#L49

I think the current dropdown does help users fill in the Windows version easily. However, for regular user, also myself only remember my Windows as either 10 or 11, without knowing the specific build like 22H2 (22621) or 21H2 (22000).

So, I still need to find out my Windows version using `Win + R` -> `winver` or `Win + Pause`. So, I think changing from a dropdown to an input field would cause some user fill windows version incorrectly but should be fine if they follow the instruction. What do you think?

Reply:
We used to use a regular input so I don't think this is an issue.